Makefile

```
@@ -0,0 +1,15 @@
+dev:
+	dotenvx run -f .env.dev -- docker compose -f docker-compose.dev.yml up -d
+	$(MAKE) keycloak-master-fix
+
+test:
+	docker compose -f docker-compose.test.yml up -d
+
+full:
+	docker compose up -d
+	$(MAKE) keycloak-master-fix
+
+.PHONY: keycloak-master-fix dev
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)